src/main/java/com/github/pjfanning/xlsx/impl/ooxml/HyperlinkData.java

```diff
@@ -35,4 +37,17 @@ public String getDisplay() {
   public String getTooltip() {
     return tooltip;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
I'm reluctant to accept this change - I think all fields should be part of the equals and hashcode not just the id and ref

Reply:
Ok, I wasn't sure what fields are unique/required, will change it